test-common/build.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
     api project(':mongo-java-server-core')
     api 'org.mongodb:mongo-java-driver:latest.release'
     api 'org.mongodb:mongodb-driver-reactivestreams:[1.13.0,2.0.0)'
+    compile group: 'io.reactivex.rxjava2', name: 'rxjava', version: '2.2.19'
```

Comment:
is it absolutely necessary to have this dependency for the test?

If it really absolutely necessary: Please don’t use the `compile` as it’s deprecated. Version should be `latest.version`. Then we can use `./updateDependencies.sh` to bump it always to the latest version.

Reply:
We need the dependency because it includes the TestSubscriber which is the class used to test with rxjava.

Reply:
No, not really. Removed the dependency in 521211563888028c7637a2060ce2bb33c7ac0a48.